Products/Zuul/facades/devicefacade.py

```diff
@@ -711,11 +711,13 @@ def addDevice(self, deviceName, deviceClass, title=None, snmpCommunity="",
                                                title=title)
         return jobrecords
 
-    def remodel(self, deviceUid):
-        fake_request = {'CONTENT_TYPE': 'xml'}
+    def remodel(self, deviceUid, collectPlugins, background):
```

Comment:
The new options should be optional keyword args so as not to break backwards compatibility or customer scripts.

Reply:
Yes sure, good point.